custom_cms.sql

```diff
@@ -3,7 +3,11 @@
 -- https://www.phpmyadmin.net/
 --
 -- Host: localhost:3306
+<<<<<<< HEAD
```

Comment:
can you remove these arrows, it is from conflicts I guess. @vbadkar

Reply:
oh, okay i'll remove it.

Reply:
@Libbna Resolved the conflct, can you check it now?